presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PushAggregationThroughOuterJoin.java

```diff
@@ -322,9 +336,21 @@ private static boolean groupsOnAllColumns(AggregationNode node, List<Symbol> col
         return Optional.of(new MappedAggregationInfo(aggregationOverNullRow, aggregationsSymbolMapping));
     }
 
+    private static OrderingScheme inlineOrderBySymbols(Map<Symbol, SymbolReference> symbolMapping, OrderingScheme orderingScheme)
```

Comment:
Add a comment to denote this logic is from `ExpressionTreeRewriter::rewriteSortItems`?
Could you also add a test to cover this code path in `TestPushAggregationThroughOuterJoin`?